snippets/card-product.liquid

```diff
@@ -82,7 +82,7 @@
         <div class="card__content">
           <div class="card__information">
             <h3 class="card__heading">
-              <a href="{{ card_product.url | default: '#' }}" class="full-unstyled-link">
+              <a href="{{ card_product.url }}" class="full-unstyled-link">
```

Comment:
Has it turned out that it's impossible for there to not be a URL in the case of a product that exists?  I had suspected it, but have you confirmed it?